bokehjs/src/coffee/renderer/guide/categorical_axis.coffee

```diff
@@ -26,6 +26,9 @@ class CategoricalAxis extends Axis.Model
 
     return range_bounds
 
+  display_defaults: () ->
+    return super()
+
```

Comment:
Sorry again. This is because the JS codes does not (at least yet) do anything like the python code does to traverse the class hierarchy. A `LinearAxis` will not get the `Axis` display defaults without this. (c.f. the `HasParent` code) I would definitely appreciate other ideas.